src/Mafin.Wizard.Api/Generator.cs

```diff
@@ -0,0 +1,64 @@
+using Mafin.Wizard.Api.Models;
+using Scriban;
+using Scriban.Runtime;
+using SharpCompress;
+
+namespace Mafin.Wizard.Api;
+
+public class Generator
+{
+    private readonly WizardSettings _settings;
+    private readonly TemplateContext _templateContext;
+    private readonly ResourceTemplateProvider _templateProvider;
+    private readonly TemplateReader _templateReader;
+
+    public Generator(WizardSettings settings)
+    {
+        _settings = settings;
+        _templateProvider = new ResourceTemplateProvider(new string[] { string.Empty, _settings.Language }, _settings.Features.ToArray());
+        _templateReader = new TemplateReader(_templateProvider);
+        _templateContext = CreateTemplateContext();
+    }
+
+    public void Create()
```

Comment:
It should return some results, let's say "Virtual Files in Memory". And then consumer will decide whether to flush it to a disk, or to a database.